src/admin/components/ResourceTabs/AuditLogTab/components/AuditLogSiteTabSelection.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from "react";
+
+import Button from "@/components/elements/Button/Button";
+
+interface AuditLogSiteTabSelectionProps {
+  buttonToogle: number;
+  setButtonToogle: (buttonToogle: number) => void;
+}
+
+const tabNames = ["Project Status", "Site Status", "Polygon Status"];
+
+const AuditLogSiteTabSelection: FC<AuditLogSiteTabSelectionProps> = ({ buttonToogle, setButtonToogle }) => {
+  return (
```

Comment:
Simple arrow functions with a single return statement should typically avoid the return to reduce indentation level
```
const AuditLogSiteTabSelection: FC<AuditLogSiteTabSelectionProps> = ({ buttonToogle, setButtonToogle }) => (
  <div ...
    ...
  </div>
);
```